Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs

```diff
@@ -330,8 +338,10 @@ public static void SetSpatialAwarenessLayer()
         /// <summary>
         /// Discover and set the appropriate XR Settings for virtual reality supported for the current build target.
         /// </summary>
+        /// <remarks>Has no effect on Unity 2020 or newer. Will be updated if a replacement API is provided by Unity.</remarks>
         public static void ApplyXRSettings()
         {
+#if !UNITY_2020_1_OR_NEWER
```

Comment:
2020.1 or 2020.2? Most of the other conditionals added are for 2020.2

Reply:
I left the ones that were compiling out/in specific APIs as `2020_2`, where the new APIs exist for the first time.

I left the ones that were compiling out functionality / legacy XR code as `2020_1`, since even those APIs still exist there (and are removed in `2020_2`), they shouldn't really be used.

I'm happy to make them all `2020_2`, but wanted to help kickstart the migration off legacy in 2020.1.

Reply:
Followed up offline. Resolving this and merging!